src/renderer/shared/util/sendAsset.js

```diff
@@ -16,17 +14,38 @@ export default async function sendAsset({ net, asset, amount, receiver, address,
     throw new Error(`Invalid amount: "${amount}"`);
   }
 
-  const selectedAsset = ASSETS[asset];
-  const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
-
-  const config = {
-    net,
-    address,
-    privateKey: new wallet.Account(wif).privateKey,
-    intents
-  };
-
-  const { response: { result, txid } } = await api.sendAsset(config, api.neoscan);
+  let result;
+  let txid;
+  if (keys(ASSETS).includes(asset)) {
+    const selectedAsset = ASSETS[asset];
+    const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
+
+    const config = {
+      net,
+      address,
+      privateKey: new wallet.Account(wif).privateKey,
+      intents
+    };
+
+    ({ response: { result, txid } } = await api.sendAsset(config, api.neoscan));
+  } else {
+    ({ response: { result, txid } } = await Neon.doInvoke({
+      net,
+      address,
+      script: createScript(asset, 'transfer', [
+        address,
+        receiver,
+        new u.Fixed8(amount)
+      ], true),
+      privateKey: wif,
+      gas: 0,
+      intents: [{
+        assetId: GAS,
+        value: '0.00000001',
+        scriptHash: wallet.getScriptHashFromAddress(address)
+      }]
+    }));
```

Comment:
This function has a bit of duplication now.  I think some of it can be prevented.  Perhaps reusing the config object is a good starting point, and maybe encapsulating the divergent code paths into a helper function would help.  e.g.:

```js
export default async function sendAsset({ net, asset, amount, receiver, address, wif }) {
  // snip...

  const send = () => {
    const config = { net, address, privateKey: wif };

    if (keys(ASSETS).includes(asset)) {
      const selectedAsset = ASSETS[asset];
      const intents = api.makeIntent({ [selectedAsset]: amount }, receiver);

      return api.sendAsset({ ...config, intents }, api.neoscan);
    } else {
      const script = createScript(asset, 'transfer', [address, receiver, new u.Fixed8(amount)], true);

      return api.doInvoke({ ...config, script, gas: 0 }, api.neoscan);
    }
  };

  const { response: { result, txid } } = await send();
 
  // snip...
}
```